### PR TITLE
Automate Compute HA scenario.

### PR DESCRIPTION
**Pull request by gosipyan**

### CODE REVIEW

File: scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+- barclamp: rabbitmq
+  attributes:
+    trove:
+      enabled: true
+    ha:
+      storage:
+        shared:
+          device: ##shared_nfs_for_rabbitmq##
+          fstype: nfs
+          options: nfsvers=3
```
Comment (kbaikov) [inline scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml]: Is nfsvers=3 still needed?

File: scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+            devices:
+            - ''
+      per_node:
+        nodes:
+          @@controller1@@:
+            params: ''
+          @@controller2@@:
+            params: ''
+          "@@controller3@@":
+            params: ''
```
Comment (kbaikov) [inline scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml]: Should controller3 be in quotes?

File: scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+- barclamp: keystone
+  deployment:
+    elements:
+      keystone-server:
+      - cluster:services
+
+- barclamp: glance
+  attributes:
+    default_store: swift
+  deployment:
```
Comment (kbaikov) [inline scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml]: Glance uses swift but swift is not installed.

File: scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml (hunk 1 above)
Comment (gosipyan) [inline scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml]: @kbaikov we have it in all our scenarios .Not sure if it change something if I remove it .

Comment (vmoravec) [inline scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml]: Please leave it, we need to test different version of nfs on our server first.

File: scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+      - @@controller1@@
+      - @@controller2@@
+      - @@controller3@@
+
+- barclamp: neutron
+  attributes:
+    ml2_type_drivers:
+    - gre
+    - vxlan
+    - vlan
```
Comment (vmoravec) [inline scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml]: Could you pls add the item `num_vlans: 10` to `attributes` section as the default number is 2000 which a bit too high for our config.

**Pull request closed without merging**